virtualmouse/fakeinput.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"github.com/BurntSushi/xgb"
+	"github.com/BurntSushi/xgb/xproto"
+	"github.com/BurntSushi/xgb/xtest"
+)
+
+type ConnInfo struct {
+	Conn  *xgb.Conn
+	Wid   xproto.Window
+	Setup *xproto.SetupInfo
+}
+
+func EstablishConn(display string) (*ConnInfo, error) {
+	X, err := xgb.NewConnDisplay(display)
+	if err != nil {
+		return nil, err
+	}
+	setup := xproto.Setup(X)
+	screen := setup.DefaultScreen(X)
+
+	if xtesterr := xtest.Init(X); xtesterr != nil {
+		return nil, xtesterr
+	}
+	return &ConnInfo{X, screen.Root, setup}, nil
+}
+
+func (c *ConnInfo) Fakeinput(x int16, y int16) error {
```

Comment:
Function name is misleading? This function moves the mouse, so just name it as that?